src/pages/Facility/settings/specimen-definitions/UpdateSpecimenDefinition.tsx

```
@@ -45,7 +47,16 @@ export function UpdateSpecimenDefinition({
     });
 
   if (isFetching) {
-    return <div>Loading...</div>;
+    return (
+      <div
+        className="p-4"
+        role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)